### PR TITLE
modify power tag display to use mini cards

### PR DESCRIPTION
**Pull request by jywarren**
Builds on part of #6593 --

Using https://getbootstrap.com/docs/4.4/components/card/

image

After, we are aiming for the badges to look like this:

image

### CODE REVIEW

**Comment by jywarren** [comment]
Needs a bit more refinement! But getting there:

image

With more spacing maybe we are good for an initial pass.

**Comment by jywarren** [comment]
ok, removed followers count from power tags

**Comment by jywarren** [comment]
```
 test_removing_tags_from_the_post#PostTest (179.08s)
Minitest::UnexpectedError:         Capybara::ExpectationNotMet: expected to find visible css ".tags-list .card-body" 2 times, found 3 matches: "awesome\n3 following\nFollow", "chapter\n1 following\nFollow", "everything"
            test/system/post_test.rb:52:in `block in <class:PostTest>'
```

**Comment by jywarren** [comment]
OK - errors:

```
 FAIL["test_adding_a_location_to_the_wiki", #<Minitest::Reporters::Suite:0x00007fcbcfa1e738 @name="PostTest">, 239.69259713200003]
 test_adding_a_location_to_the_wiki#PostTest (239.69s)
        expected to find css ".tags-list .badge a[href=\"/tag/lat:22\"]" but there were no matches
        test/system/post_test.rb:210:in `block in <class:PostTest>'


=======[Screenshot]: tmp/screenshots/failures_test_removing_tags_from_the_post.png
ERROR["test_removing_tags_from_the_post", #<Minitest::Reporters::Suite:0x00007fcbcee37708 @name="PostTest">, 329.98693319999995]
 test_removing_tags_from_the_post#PostTest (329.99s)
Minitest::UnexpectedError:         Capybara::ElementNotFound: Unable to find css ".tags-list p.badge .tag-delete"
            test/system/post_test.rb:57:in `block (2 levels) in <class:PostTest>'
            test/system/post_test.rb:56:in `block in <class:PostTest>'
```

File: app/views/tag/_miniCard.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="card" style="clear:left;">
+<div class="card<% if local_assigns[:hide_button] %> more-tags<% end %>" style="clear:left;<% if local_assigns[:hide_buttong] %>display:none;<% end %><% if local_assigns[:grey] %>background:#eee;<% end %>">
```
Comment (jywarren) [inline app/views/tag/_miniCard.html.erb:1]: we could redo the background here using css and then system test for it...

**Comment by jywarren** [comment]
OK cool - so on adding tags, they don't yet display in cards, so we are seeing this first error:

```

 test_adding_tags_to_the_post#PostTest (166.06s)
Minitest::UnexpectedError:         Capybara::ExpectationNotMet: expected to find css ".tags-list .card" 2 times but there were no matches
            test/system/post_test.rb:43:in `block in <class:PostTest>'
```

Then, probably we have to update the selectors to the card-style deletion buttons:

```
==[1105/201552.271467:WARNING:spdy_session.cc(3384)] Received HEADERS for invalid stream 121
====[Screenshot]: tmp/screenshots/failures_test_removing_tags_from_the_post.png
ERROR["test_removing_tags_from_the_post", #<Minitest::Reporters::Suite:0x00007f4abc139a30 @name="PostTest">, 254.78303012099997]
 test_removing_tags_from_the_post#PostTest (254.78s)
Minitest::UnexpectedError:         Capybara::ExpectationNotMet: expected to find visible css ".tags-list .card" 0 times, found 3 matches: "awesome\n3 following\nFollow", "chapter\n1 following\nFollow", "everything"
            test/system/post_test.rb:54:in `block in <class:PostTest>'
```

**Comment by jywarren** [comment]
Great, next we'll get the tag display for JS/Ajax inserted tags to be a card, not a badge: 

https://github.com/publiclab/plots2/blob/b007b810b9fa62e48d40209cbdeb95625dcc2bfd/app/assets/javascripts/tagging.js#L95-L101

Could we load it as a partial, from Rails, maybe, so it's identical, and not duplicated code? It'd take a new controller, i think.

**Comment by ebarry** [comment]
excited for this to merge!

**Pull request closed without merging**